opencti-platform/opencti-front/src/components/ItemOpenVocab.tsx

```diff
@@ -22,12 +22,28 @@ const useStyles = makeStyles(() => ({
   smallIcon: {
     margin: '5px 0 0 10px',
   },
+  inlineIcon: {
```

Comment:
I needed a style to display the reliability right next to the confidence, and since there is not much space, I added this style "inline" which displays it with less margin.